pom.xml

```diff
@@ -1036,6 +1037,7 @@
                         <!-- set heap size to work around https://github.com/travis-ci/travis-ci/issues/3396 -->
                         <argLine>-Xmx3000m -Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8
                             -Duser.timezone=UTC -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager
+			    -Ddruid.null.handling.useDefaultValueForNull=${druid.null.handling.useDefaultValueForNull}
```

Comment:
Couldn't set `-D` param from CI launch line directly, without adding a property to maven?